include/deal.II/lac/la_parallel_vector.h

```diff
@@ -244,7 +244,7 @@ namespace LinearAlgebra
       /**
        * Destructor.
        */
-      virtual ~Vector ();
+      ~Vector () override;
```

Comment:
I think we should keep the destructor `virtual` here. Also strange to have `override` for a destructor.